app/styles/alert.less

```diff
@@ -0,0 +1,81 @@
+.upf-alert-v2 {
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  overflow: hidden;
+  border: 1px solid var(--color-border-default);
+  border-radius: var(--border-radius-md);
+
+  .main-container {
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+    padding: @spacing-xx-sm;
+    gap: @spacing-xx-sm;
+    background-color: var(--color-gray-50);
+  }
+
+  .text-container {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    gap: 3px;
```

Comment:
This is to "skip" the position of the "indicator" element right ?
i suggest you define a local variable for this ; it would make more sense while reading the code.

Reply:
No, this is just the gap between title and subtile. I can set it in variable, no problem with that